app/queries/crash.ts

```diff
@@ -129,7 +129,7 @@ export const GET_CRASH = gql`
           sus_serious_injry_count
         }
       }
-      people_list_view {
+      people_list_view (order_by: { unit_nbr: asc }) {
```

Comment:
i noticed the sort order of the rows jumping around while editing. would you add sorting on `prs_nbr` ascending as well? so that the records are sorted by unit # then person #?

Reply:
it also looks like you need to add the foreign key columns for all the editable fields with lookup values, e.g., `prsn_injry_sev_id`. 

making this change will ensure that the correct option is initially selected when you begin editing a lookup field. otherwise it defaults to "Select..." 👇 

<img width="400" alt="Screenshot 2024-12-31 at 11 11 34 AM" src="https://github.com/user-attachments/assets/85664b08-e390-42d2-aaac-4d7bd96924c2" />


And [here](https://github.com/cityofaustin/vision-zero/pull/1634#discussion_r1894241301) is a comment where i mention that this is not well documented 😇 any ideas?